jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -399,6 +399,9 @@ void jbmc_parse_optionst::get_command_line_options(optionst &options)
       }
     }
   }
+
+  if(cmdline.isset("java-threading"))
+    options.set_option("java-threading", true);
```

Comment:
I would like the option being used by goto-symex to be more general (essentially, "java" should not appear anywhere in goto-symex as it's supposed to be a general tool). You might pick "allow-pointer-unsoundness" or the likes.

Reply:
done, see  https://github.com/diffblue/cbmc/pull/2354